src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/PlaceholderAdminNode.Fields.TreeEdit.cshtml

```diff
@@ -64,6 +66,59 @@
     </div>
 </div>
 
+<div class="form-group">
+    @{
+        var selectedItems = JsonConvert.SerializeObject(Model.SelectedItems, new JsonSerializerSettings { ContractResolver = new CamelCasePropertyNamesContractResolver() });
+        var allItems = JsonConvert.SerializeObject(Model.AllItems, new JsonSerializerSettings { ContractResolver = new CamelCasePropertyNamesContractResolver() });
+    }
+
+    <script asp-name="vue-multiselect-wrapper" asp-src="~/OrchardCore.AdminMenu/Scripts/vue-multiselect-wrapper.js" at="Foot" depends-on="vuejs, vue-multiselect, sortable, vuedraggable"></script>
```

Comment:
We shouldn't use the same name here.
It won't matter much on this page, but it's technically a conflict with the content fields resource name.

So should be named differently.